scripts/rust-analyze.sh

```diff
@@ -49,6 +51,11 @@ if [ -d "$OBJDIR" ]; then
   if [ -d "$INDEX_ROOT/rustlib" ]; then
     ANALYSIS_DIRS="$ANALYSIS_DIRS $(find $INDEX_ROOT/rustlib -type d -path \*/${PLATFORM_FILTER}analysis)"
   fi
+  # If a sed normalization command was provided, perform the transform over all
+  # json analysis files.
+  if [[ $SED_TRANSFORM_EXPR ]]; then
+    find $ANALYSIS_DIRS -type f -name "*.json" | parallel --halt now,fail=1 sed --in-place -e "'$SED_TRANSFORM_EXPR'" {}
```

Comment:
I still find it weird that we need two sets of quotes here, but maybe `parallel` is munging things. I guess this is fine.